cfpb/cfgov-refresh
# Job posting location test

We want to run a quick test on job aggregators' ability to read location data before our current job postings close early next week by adding a temporary hard-coded section to the job posting template

## Additions

- Temporary hard-coded list of cities associated with a job region

## Testing

1. Check out branch
2. In admin, create (or update) a job posting in one of the 4 multi-state regions (West, Midwest, Southeast, Northeast)
3. Navigate to job posting page and verify in dev tools that a hidden "location" section appears after the region section at the top of the page


## Screenshots

<img width="987" alt="screen shot 2017-12-13 at 11 26 42 am" src="https://user-images.githubusercontent.com/778171/33958163-882d3a4e-dff8-11e7-9d42-1f4429cc3ff3.png">

## Checklist
